login_fallback.rs

```diff
@@ -0,0 +1,576 @@
+pub mod login_fallback {
```

Comment:
I'm guessing this file wasn't supposed to be committed? ;)

Reply:
It's okay, I've only done that mistake twice today :smiling_face_with_tear:

Reply:
Removed